autoscheduler/frontend/src/components/SchedulingPage/Schedule/MeetingCard/MeetingCard.tsx

```diff
@@ -44,7 +44,18 @@ const MeetingCard: React.FC<MeetingCardProps> = ({
       borderColor={bgColor}
     >
       <Typography variant="body2" data-testid="meeting-card-primary-content">
-        {`${section.subject} ${section.courseNum}-${section.sectionNum}`}
+        {`${section.subject} ${section.courseNum}`}
+        {isBig
+          ? (
+            <Typography variant="body2" component="span">
+              {`-${section.sectionNum}`}
+            </Typography>
+          )
```

Comment:
```suggestion
          ? `-${section.sectionNum}`
```
Since this is the same variant as the outer typography, you don't need to make another component